src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -16,6 +16,12 @@ class ConfirmationBox extends React.Component {
     this.props.history.push('/editSnack');
   };
 
+  componentDidMount() {
+    this.itemImage = this.props.storeList.find(
```

Comment:
Are the props guaranteed not to change (e.g. name, storeList)?

Reply:
shouldn't do